packages/react-browser-pathname/src/context.ts

```diff
@@ -1,7 +1,14 @@
 import React, { useContext, useCallback, useEffect, useState } from 'react';
 import type { ReactNode } from 'react';
 
+export type SET_ACTION = {
+  url?: string;
```

Comment:
why is this allowed to be optional?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/History/pushState i dunno man

Reply:
i think it might be for setting state without changing path

Reply:
" If this parameter isn't specified, it's set to the document's current URL." yup - useful for state